ModelReplacementAPI/BodyReplacementBase.cs

```diff
@@ -415,16 +410,9 @@ private void Update()
             {
                 if((bodyReplacement == null) && (modelObj != null))
                 {
-                    //Console.WriteLine("Detected floater");

```

Comment:
This is an in progress testing of how I believe floating model replacements with no player can be removed. It isn't important to remove at this moment as that testing is not concluded